src/python/impactx/MADXParser.py

```diff
@@ -0,0 +1,382 @@
+# Author: Matthias Frey, Andreas Adelmann
+# Date: 13. Oct. 2017
+#
+# This file parses MAD-X file into a beamline
+# composed of pyAcceLEGOrator elements.
+#
+# 10.8.2022 Adapted for standalone use
+#
```

Comment:
Adjust as following, please clarify with Andreas if we can add a simple `BSD-3-Clause` license. Use international date formatting:
```suggestion
#!/usr/bin/env python3
#
# Copyright 2022 ImpactX contributors
# Authors: Matthias Frey, Andreas Adelmann, Marco Garten
# License: BSD-3-Clause
#
# Changelog:
#   Oct 13th, 2017: original version for pyAcceLEGOrator elements
#   Aug 10th, 2022: adapted for standalone use
#
# -*- coding: utf-8 -*-
```